reportportal_client/core/worker.py

```diff
@@ -43,22 +43,16 @@ def is_stop_cmd(self):
 class APIWorker(object):
     """Worker that makes non-blocking HTTP requests to the Report Portal."""
 
-    def __init__(self, cmd_queue, data_queue):
-        """Initialize instance attributes.
-
-        :param cmd_queue:  Queue for the control commands
-        :param data_queue: Queue for the RP requests to process
-        """
-        self._cmd_queue = cmd_queue
-        self._data_queue = data_queue
+    def __init__(self):
+        """Initialize instance attributes."""
+        self._queue = queue.Queue()
```

Comment:
We need the PriorityQueue here. Our requests have their own priorities. Revert the changes and modify the code to use only one queue instead of two. The queue should be passed into the constructor.

Reply:
Maybe I messed that, but I don't see where we use any feature of PriorityQueue?